TI/About.html

```diff
@@ -0,0 +1,188 @@
+            </ul>
+          </div>
+        </div>
+      </div>
+      <div class="container">
+        <h2>Education</h2>
+        <hr>
+        <div class="row">
+          <div class="col-md-6">
+            <h3>Cégep Garneau</h3>
+            <p>AEC en développement web</p>
+          </div>
+        <div class="row">
+          <div class="col-md-6">
+            <h3>CFPPA</h3>
+            <p>DEP en électricité</p>
+          </div>
+          <div class="col-md-6">
+            <h3>Cégep RDL</h3>
+            <p>DEC Sciences humaines</p>
+          </div>
+        </div>
+      </div>          
+    <div class="container">
+        <h2>Hobbies</h2>
+        <ul class="list-group">
+          <li class="list-group-item">
+            <div class="row">
+              <div class="col-md-2">
+                <img src="./assets/cook.jpg" width="150px" height="150px" class="img-fluid" alt="cuisine">
+              </div>
+              <div class="col-md-10">
+                <h3>Cuisiner</h3>
+                <p>J'adore cuisiner car j'aime surveiller ce que je mange et apprendre a agencer les goûts.</p>
+              </div>
+            </div>
+          </li>
+          <li class="list-group-item">
+            <div class="row">
+              <div class="col-md-2">
+                <img src="./assets/lift.jpg" width="150px" height="150px" class="img-fluid" alt="gym">
+              </div>
+              <div class="col-md-10">
+                <h3>Conditionnement physique</h3>
+                <p>J'aime me surpasser à la salle pour devenir la meilleure version de moi-même et être en santé.</p>
+              </div>
+            </div>
+          </li>
+          <li class="list-group-item">
+            <div class="row">
+              <div class="col-md-2">
+                <img src="./assets/anime.png"  width="150px" height="150px" class="img-fluid" alt="anime">
+              </div>
+              <div class="col-md-10">
+                <h3>Anime</h3>
+                <p>J'aime me divertir avec des séries animés sans prise de tête dans mes temps libres.</p>
+              </div>
+            </div>
+          </li>
+          <li class="list-group-item">
+            <div class="row">
+              <div class="col-md-2">
+                <img src="./assets/golf.jpg" width="150px" height="150px" class="img-fluid" alt="golf">
+              </div>
+              <div class="col-md-10">
+                <h3>Golf</h3>
+                <p>Le golf demande une bonne discipline et beaucoup de patience en plus de pouvoir passer du bon temps dehors avec des amis!</p>
+              </div>
+            </div>
+          </li>
+        </ul>
+      </div>             
+</body>
+<footer class="bg-light">
+  <div class="container py-3">
+      <div class="row">
+          <div class="col-md-6">
+              <a class="nav-link" href="https://www.facebook.com/profile.php?id=100016978482689">Facebook</a>

```

Comment:
Usually, external links must use `target='_blank'`. This way, the external page will be opened in a new tab, not replacing your current page.